packages/neon/neon/lib/src/settings/models/storage.dart

```diff
@@ -36,7 +36,6 @@ enum StorageKeys implements Storable {
   final String value;
 }
 
-@internal
```

Comment:
Is this really needed?

The test calls `runNeon` wich should setup the sharedPrefs with the same `await SharedPreferences.getInstance()` you are now doing.

I initally only added this function so we can pass a `Mock` object (from mocktail) into it just to unit test the `SingleValueStorage` and `AppStorage`. I don't think it's needed outside of this.